docker-compose.yml

```diff
@@ -2,16 +2,17 @@ services:
   fund-store:
     build: 
       context: ../funding-service-design-fund-store
-    command: ['flask', 'run', '--host', '0.0.0.0', '--port', '8080']
+    command: ["sh", "-c", "pip install debugpy -t /tmp && python /tmp/debugpy --listen 0.0.0.0:5678 -m flask run --no-debugger --host 0.0.0.0 --port 8080"]   
```

Comment:
could we include debugpy in the dev-requirements instead?

Reply:
Yeah, good shout. I'll work thru them and add it 👍